tests/gdx-tests-android/assets/data/maps/tiled/super-koalio/level1.tmx

```diff
@@ -16,7 +16,7 @@
   </data>
  </layer>
- <imagelayer name="koalio-color" x="352" y="136" opacity="0.5">
+ <imagelayer name="koalio-color" x="352" y="136" opacity="0.5" visible="0">
```

Comment:
Sorry, I didn't investigate properly.
Are the other changes fine? I could submit the revised commit in this case.